<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::ContentBlockingProfileDialog</class>
 <widget class="QDialog" name="Otter::ContentBlockingProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="settingsLayout">
       <item>
        <layout class="QFormLayout" name="settingsTopLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>Title:</string>
           </property>
           <property name="buddy">
            <cstring>titleLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="titleLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="categoryLabel">
           <property name="text">
            <string>Category:</string>
           </property>
           <property name="buddy">
            <cstring>categoryComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="categoryComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="cosmeticFiltersLabel">
           <property name="text">
            <string>Cosmetic filters:</string>
           </property>
           <property name="buddy">
            <cstring>cosmeticFiltersComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cosmeticFiltersComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="enableWildcardsCheckBox">
         <property name="text">
          <string>Enable wildcard expressions</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="settingsBottomLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="updateUrlLabel">
           <property name="text">
            <string>Address:</string>
           </property>
           <property name="buddy">
            <cstring>updateUrLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="updateUrLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="updateIntervalLabel">
           <property name="text">
            <string>Update interval:</string>
           </property>
           <property name="buddy">
            <cstring>updateIntervalSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="updateIntervalSpinBox">
           <property name="specialValueText">
            <string>Never</string>
           </property>
           <property name="suffix">
            <string> days</string>
           </property>
           <property name="maximum">
            <number>365</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lastUpdateLabel">
           <property name="text">
            <string>Last update:</string>
           </property>
           <property name="buddy">
            <cstring>lastUpdateTextLabel</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="updateLayout" stretch="1,0">
           <item>
            <widget class="Otter::TextLabelWidget" name="lastUpdateTextLabel" native="true"/>
           </item>
           <item>
            <widget class="QPushButton" name="updateButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Update</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sourceTab">
      <attribute name="title">
       <string>Source</string>
      </attribute>
      <layout class="QVBoxLayout" name="sourceLayout">
       <item>
        <widget class="Otter::PassiveNotificationWidget" name="passiveNotificationWidget" native="true"/>
       </item>
       <item>
        <widget class="Otter::SourceEditWidget" name="sourceEditWidget">
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="saveLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="confirmButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::PassiveNotificationWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/PassiveNotificationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Otter::SourceEditWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>src/ui/SourceEditWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::TextLabelWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/TextLabelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>confirmButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>Otter::ContentBlockingProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>Otter::ContentBlockingProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
